documentation/modules/cruise-control/con-optimization-proposals.adoc

```diff
@@ -274,3 +274,30 @@ Consider a shorter interval for fast changing clusters, although this increases
 * xref:con-optimization-goals-{context}[]
 * xref:proc-generating-optimization-proposals-{context}[]
 * xref:proc-approving-optimization-proposal-{context}[]
+
+== Auto-approving an optimization proposal
+
+In some cases, the manual approval process of the optimization proposal is not necessary.
```

Comment:
Thsi sounds weird without listing the cases where it is not necessary. I would maybe phrase it as _if you want, you can have it automatically approved_.